config/lintStaged/lintStagedConfig.js

```diff
@@ -0,0 +1,20 @@
+const { isYarn } = require('../../context');
+
+const steps = {};
+
+// Yarn lock integrity check
+if (isYarn) {
+  steps['+(package.json|yarn.lock)'] = [
+    'yarn check-integrity',
+    'git add package.json yarn.lock',
+  ];
+}
+
+// Format & lint
+steps['**/*.{js,jsx,ts,tsx,md,less,css}'] = [
```

Comment:
Should we include JSON files too?

Reply:
I think we should, the reason i didnt was to keep it in sync with the [defaults in our `runPrettier` script](https://github.com/seek-oss/sku/blob/f4cfb5ac540f0cd0acdd6e5f1383877d1b577303/lib/runPrettier.js#L40). Up for grabs is whether we should sync these and maybe import one from the other.

Thoughts?